ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -547,6 +548,12 @@ public override void ReceiveCommand(ReactTextBox view, int commandId, JArray arg
             if (commandId == FocusTextInput)
             {
                 view.Focus(FocusState.Programmatic);
+                
+                // Open the touch keyboard if the device is in touch mode
+                if (UIViewSettings.GetForCurrentView().UserInteractionMode == UserInteractionMode.Touch)
```

Comment:
I can't really speak for UWP apps in general, but I would like to give some input to keep the discussion going so we can resolve this.

Before using RN Windows, I developed apps natively and with RN for Android. Since most of these apps run on devices without a physical keyboard, the onscreen keyboard is always opened when an input gets focus. This especially makes sense with forms in a kiosk application (as I am developing now) so the user doesn't have to tap on the input field to begin typing. This small detail make the flow so much better.

However, after trying the tablet mode on Windows, the default behavior seems to be different. When opening an app or even the Windows search, the input field is focused but the keyboard stays shut. I assume, this is due to the complexity of these apps and all the possible interactions the user can do. So I think the default behavior of UWP apps might be to keep the keyboard hidden until the use manually taps an input. This matches the current behavior.

Since both of these flows make sense, I see two possible solutions:
1. make the behavior similar to Android & iOS and always the deploy the touch keyboard when in tablet mode. This would RN Windows more similar to RN but different from native apps & the OS
2. add an extra command to deploy the keyboard programmatically. This would keep the default behavior similar to other UWP apps & the OS but add a knob for special use cases where automatically showing the touch keyboard is desired.

Reply:
In tablet mode, when XAML's textbox gets focus SIP will deploy automatically, that's the default mode. But some app (i.e. Settings) does not want that default behavior since the SIP will take away large part of available UI space, so they customized the TextBox with TextBox.PreventKeyboardDisplayOnProgrammaticFocus set to true. This means SIP will not deploy for programmatically set focus such as app start. User needs to tap into TextBox to bring up the SIP.